CARDS-1677: Update DATA-PRO TOU

k-pace opened the pull request.

Commit:
Updated TOU

Commit:
Added Privacy Policy

marta- commented on proms-resources/clinical-data/src/main/resources/SLING-INF/content/Homepage/Proms/TermsOfUse.xml:
```diff
@@ -21,146 +21,168 @@
-These terms and conditions (the "Agreement") for a symptoms survey located at datapro.uhn.ca and any associated website, portal, or similar service (collectively, the "Application"), constitute a legal agreement and is entered into by and between you ("you", "your") and the University Health Network ("UHN", "we", "us", "our").  
 
-**IMPORTANT! BY USING OR ACCESSING THE APPLICATION, YOU AGREE TO BECOME BOUND BY THE AGREEMENT. IF YOU DO NOT AGREE WITH THE AGREEMENT, YOU MUST NOT USE OR ACCESS THE APPLICATION. NOTE THAT THERE ARE ALTERNATIVE METHODS TO COMPLETE THE SURVEY IF YOU WOULD NOT LIKE TO PROCEED WITH THIS APPLICATION.**  
+## ePRO TERMS OF USE
```
The title property from line 24 is displayed in the ToU dialog title bar. Do we need to repeat it here in the text?
```suggestion
```

marta- replied:
Nevermind, turns out the text also contains Privacy policy.

marta- commented on proms-resources/clinical-data/src/main/resources/SLING-INF/content/Homepage/Proms/TermsOfUse.xml:
```diff
@@ -21,146 +21,168 @@
+### 4. How we Protect your Privacy
 
-b) **Transmission of anonymized or aggregated data to Matomo Analytics.** The Application collects data, including Personal Health Information (PHI), through the Application regarding the use of the Application in order to better understand use of the Application and to help us identify areas for improvement. If you do not consent to the transmission of this data, then you must not use the Applic ...
+In collecting and using your personal health information in ePRO, UHN must follow the rules set out in the Personal Health Information Protection Act. UHN must also follow the rules set out in the Freedom of Information & Protection of Privacy Act, which applies to administrative type information that does not identify you (such as the number of users). 
```
`&` makes the xml invalid ("Unterminated entity ref"). Needs to be escaped as `&amp;` or `&#038;`.
```suggestion
In collecting and using your personal health information in ePRO, UHN must follow the rules set out in the Personal Health Information Protection Act. UHN must also follow the rules set out in the Freedom of Information &amp; Protection of Privacy Act, which applies to administrative type information that does not identify you (such as the number of users). 
```

marta- commented on proms-resources/clinical-data/src/main/resources/SLING-INF/content/Homepage/Proms/TermsOfUse.xml:
```diff
@@ -21,146 +21,168 @@
     <property>
         <name>title</name>
-        <value>Terms of Use</value>
+        <value>ePRO Terms of Use</value>
```
```suggestion
        <value>ePRO Terms of Use and Privacy Policy</value>
```

k-pace committed:
Fixed unescaped character

Co-authored-by: Marta Girdea <marta@phenotips.org>

k-pace committed:
Updated Title

Co-authored-by: Marta Girdea <marta@phenotips.org>

marta- committed:
CARDS-1677: Formatting fixes

* De-capitalized the two main titles (ePro Terms of Use and DATA-PRO Privacy Policy)
* Fixed new lines inside addresses
* Minor spacing cleanup in ordered list

marta- commented:
@sashaandjic the bullets are not that easy to change, plus it's a convention to use different types of bullets for sublists. I would leave it as is.

marta- commented:
Before we change the main title size, I feel like not all the content that _is_ displalyed here _should_ be here.

@k-pace Does it make more sense to split this document in 2: Terms of Use, that the user can accept, and Privacy Policy, linked separately in the footer, that they can just reed through if they wish?

marta- committed:
CARDS-1677: Updated the ToU link text to include Privacy Policy

Before: "Terms of Use"
After: "Terms of Use and Privacy Policy"

marta- committed:
CARDS-1677: Increased the font size in the ToU dialog title
To make that possible, added support for specifying a `className` for the ResponsiveDialog component

marta- commented:
@sashaandjic Changes in the last 3 commits:
* Increased the font size in the ToU dialog title
* De-capitalized the two main subtitles in the text (ePro Terms of Use and DATA-PRO Privacy Policy)
* Fixed new lines inside addresses
* The "Terms of Use" link in the patient UI footer now reads "Terms of Use and Privacy policy"

marta- committed:
CARDS-1677: Minor formatting fixes

* Addresses: made indentation and bold formatting of the institution name consistent across all addresses
* Split Copyright and Date Last Revised on separate lines
* Added a horizontal divider between ToU and PP sections

sashaandjic reviewed (approved):
tested

sashaandjic commented:
QUESTION: The AUDIT-C form got generated and marked as completed, even though I did not touch it.
![image](https://user-images.githubusercontent.com/23243244/156668888-2def7c22-a358-4fba-bb6a-3b6009e1ba19.png)

When I opened the form it was completely blank. Then I just click on the edit button and did nothing. Now form looks the way I would expect it to look.
![image](https://user-images.githubusercontent.com/23243244/156669110-2913de17-463e-4a11-832a-1b43d6f495b6.png)

sashaandjic commented:
all good now (except the fact that I had to use -Pquick when making a build)
![image](https://user-images.githubusercontent.com/23243244/156677299-4562241c-05b2-482c-b0e8-98ef223bf1c4.png)

sashaandjic reviewed (approved):
tested

sashaandjic commented:
Please note that on a second attempt I was able to make a build without using -Pquick option.
